src/components/join/How.jsx

```diff
@@ -1,7 +1,75 @@
+"use client";
+import Image from "next/image";
+import image1 from "@/public/join/how.webp";
+import Link from "next/link";
+import useView from "../useView";
+
 const How = () => {
   return (
-    <div className="">
-      <p>How</p>
+    <div className="ml-7">
+      <p
+        className={`${
+          useView && "animate-fade-right animate-delay-[100ms]"
+        }text-2xl md:text-4xl text-swim-blue-400 font-bold mb-10 `}
+      >
+        How To Practice and Officially become a member:
+      </p>
+      <div className="md:flex md:flex-row">
+        <div
+          className={`${
+            useView && "animate-fade-right animate-delay-[150ms]"
+          } md:flex md:flex-col items-center`}
```

Comment:
```suggestion
          } md:flex md:flex-row items-center`}
```

when you switch instructions 1 & 2